website/docs/artist_hosts_nuke_tut.md

```diff
@@ -0,0 +1,166 @@
+
+This QuickStart is just a small introduction to what OpenPype can do for you. It attempts to make an overview for compositing artists, and simplifies processes that are better described in specific parts of the documentation.
+
+## Run Nuke - Shot and Task Context
+OpenPype has to know what shot and task you are working on. You need to run Nuke in context of the task, using Ftrack Action or OpenPype Launcher to select the task and run Nuke.
+
+![Run Nuke From Ftrack](assets/nuke_tut/nuke_RunNukeFtrackAction_p2.png)
+![Run Nuke From Launcher](assets/nuke_tut/nuke_RunNukeLauncher_p2.png)
+
+
+OpenPype menu shows the current context - shot and task.
+
+If you use Ftrack, executing Nuke with context stops your timer, and starts the Ftrack clock on the shot and task you picked.
+
+![Context](assets/nuke_tut/nuke_Context.png)
+
+:::tip Admin Tip - Nuke version
+You can [configure](admin_settings_project_anatomy.md#Attributes) which version(s) will be available for current project in **Studio Settings → Project → Anatomy → Attributes → Applications**
+:::
+
+## Save Nuke script – the Work File
+Use OpenPype - Work files menu to create a new Nuke script. Openpype offers you the preconfigured naming. 
+![Context](assets/nuke_tut/nuke_AssetLoader.png)
+
+Openpype makes initial setup for your Nuke script. It is the same as running [Apply All Settings](artist_hosts_nuke.md#apply-all-settings) from the OpenPype menu.
+Reads frame range and resolution from Avalon database, sets it in Nuke Project Settings
+Creates Viewer node, sets it’s range and indicates handles by In and Out points
+Reads Color settings from the project configuration, and sets it in Nuke Project Settings and Viewer
+Sets project directory in the Nuke Project Settings to the Nuke Script Directory
+
+:::tip Admin Tip - Workfile Naming
+The [workfile naming](admin_settings_project_anatomy#templates) is configured in anatomy, see **Studio Settings → Project → Anatomy → Templates → Work**
+:::
+
+:::tip Admin Tip - Open Workfile
+You can [configure](project_settings/settings_project_nuke#create-first-workfile) Nuke to automatically open the last version, or create a file on startup. See **Studio Settings → Project → Global → Tools → Workfiles**
+:::
+
+:::tip Admin Tip - Nuke Color Settings
+[Color setting](project_settings/settings_project_nuke) for Nuke can be found in **Studio Settings → Project → Anatomy → Color Management and Output Formats → Nuke**
+:::
+
+## Load plate – Asset Loader
+If your IO or editorial prepared plates and references, or your CG team has a render to be composited, we need to load it.
+
+![OpenPype Load](assets/nuke_tut/nuke_Load.png)
+![Asset Load](assets/nuke_tut/nuke_AssetLoader.png)
+
+Pick the plate asset, right click and choose Load Image Sequence to create a Read node in Nuke. Note that the Read node created by OpenPype is green and has an OpenPype Tab. Green color indicates the highest version available.
```

Comment:
We could be explained the differences between Load Sequences/Image/video plugins and how to use them. This is just an idea - I understand this should be quick start tutorial so perhaps link to some other page which is explaining those loaders will work.

Reply:
Had to think twice what "the highest version available" actually means. I would use:
" Green color indicates the highest version of asset is loaded. Asset versions could be easily changed by Manage (described later). In that case lower versions will be highlighted by orange color on the read node."

Reply:
@jezscha I will just add link to Asset Loader docs for now - artist_tools#loader
@kalisp Thank you, will add this